docs/development/coding_guides/css_guidelines.rst

```diff
@@ -29,73 +29,63 @@ designers sometimes have very different angles on the same things.
 The following terminology is therefore based on the tried and tested
 systems mentioned above.
 
-We think of elements in the user interface (UI) as *objects*. A typical
-object might be a link, button or a list. It is important to understand
-the difference between a specific object (e.g. "the safe button in the
-login view") and its *class* ("button").
-
-We distinguish several types of objects. There are also the related
-concepts *state*, *modifier*, *variable*, *mixin*, and *adjustment*.
-
-Base Objects
+Base Styling
 ++++++++++++
 
-When HTML elements are used without any special modifications, they are called
-base objects. The basic styling of these elements sets the prevailing mood for
-a product. This involves general *text styling* as well as *links*, *headings*,
-and *input boxes*.
+Base styling is the styling that applies to HTML elements when no
+additional classes are added. It sets the prevailing mood for a product.
+This involves general *text styling* as well as *links*, *headings*, and
+*input boxes*.
 
-Widget Objects
-++++++++++++++
-Widgets are the most common objects. They can be reused throughout the UI. A
-typical widget is a *button* or a *login dialog*. While base objects are
-limited to existing HTML elements we can define our own widgets.
+Layout
+++++++
 
+The layout defines the position of elements on a page. It is typically
+based on a grid system.
 
-Layout Objects
-++++++++++++++
+Components
+++++++++++
 
-The layout defines the basic structure of a page. Typical layout objects
-are *header*, *main*, *aside*, and *footer*. Of most layout objects there
-is only a single instance on a page (e.g. only one header and footer).
+There are many synonyms for this in the different methodologies:
+Object (OOCSS), module (Smacss), block (BEM), atom/molecule/organism
+(Atomic Design), or pattern (pattern driven design).
```

Comment:
I cannot remember where this term came from and it is not used in any of the referenced systems. Feel free to add it to the list if you feel it is relevant.